### ISSUE TITLE
Contrast fail on tabs

### ISSUE THREAD

**Issue by Dawdre** [alphagov/govuk-frontend#1950]
Currently the contrast between non selected tabs and a white background is failing all WCAG checks. This was raised as part of a 3rd party accessibility audit carried out on one of our applications recently.

![image](https://user-images.githubusercontent.com/11505395/92604359-5bfdab00-f2a8-11ea-8ff1-00bb4b76ae58.png)

sidenote: #F3F2F1 looks like #FFFFFF when viewing through Windows RDP. A little heads up to anyone WFH through an RDP during these times. It caused me some confusion for far longer than it should of! 😄

**Comment by timpaul** [comment]
Thanks for the feedback @Dawdre. 

We added underlines to the text of unselected tabs, so that users would still understand that they were clickable even if they couldn't perceive the tab boundaries.

We believe that this meets the relevant success criterion of WCAG 2.1 ([Non-text contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)) - but please let us know if you see it causing issues in user research.

The issue with visual compression of remote desktops is an interesting one. It was [a discussion about this issue](https://github.com/alphagov/govuk-design-system-backlog/issues/100#issuecomment-460969754) that prompted us to add the underlines - but I guess a lot more people are using RDP now...
